templates/invite.hamlet

```diff
@@ -1,4 +1,6 @@
 $maybe link <- maybe_link
+    <h1 .text-center>
+        Invite
```

Comment:
Instead of the duplicate titles within the $maybe and the $nothing below, just put a single title above the $maybe